bouncer-log-analysis.py

```diff
@@ -497,9 +497,12 @@ def plot_rolling_request_rate(df, matcher):
     plt.ylabel('Amplitude')
     ax.set_xscale('log')
     set_title(matcher.description)
-    subtitle = 'Freq spec from narrow rolling request rate -- ' + \
-        matcher.subtitle
-    set_subtitle(subtitle)
+    try:
+        subtitle = 'Freq spec from narrow rolling request rate -- ' + \
+            matcher.subtitle
+        set_subtitle(subtitle)
+    except (TypeError):
+        pass
```

Comment:
The right thing to do is to make the `--subtitle` argument required instead of optional. I am merging yours and change this in a follow-up patch.